client/src/containers/ForksPage/index.jsx

```diff
@@ -24,103 +24,116 @@ class ForksPage extends Component {
 
   getForksList() {
     const { repositoryId } = this.props;
-    getForksList(repositoryId).then(forks => {
-      this.setState({ forks, loading: false });
+    getForksList(repositoryId).then(async forks => {
```

Comment:
`this.setState` doesn't return Promise, so `await` has no effect here. Use setState's callback if you need to execute something after setting of state.
```suggestion
    getForksList(repositoryId).then(forks => {
      this.setState({ forks, loading: false });
```